src/Microsoft.VisualStudio.Web.BrowserLink/SendFilesWrapper.cs

```diff
@@ -1,38 +1,59 @@
 ﻿using System.IO;
+using System.IO.Pipelines;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 
+
 namespace Microsoft.VisualStudio.Web.BrowserLink
 {
-    internal class SendFilesWrapper : IHttpSendFileFeature
+    internal class SendFilesWrapper : IHttpResponseBodyFeature
     {
         private HttpResponse _response;
-        private IHttpSendFileFeature _wrapped;
+        private IHttpResponseBodyFeature _wrapped;
 
-        internal SendFilesWrapper(IHttpSendFileFeature wrapped, HttpResponse response)
+        internal SendFilesWrapper(IHttpResponseBodyFeature wrapped, HttpResponse response)
         {
             _wrapped = wrapped;
             _response = response;
         }
 
-        async Task IHttpSendFileFeature.SendFileAsync(string path, long offset, long? count, CancellationToken cancellation)
+        public Stream Stream => _wrapped.Stream;
+
+        public PipeWriter Writer => _wrapped.Writer;
+
+        public Task CompleteAsync()
+        {
+            return _wrapped.CompleteAsync();
```

Comment:
It used to be possible for _wrapped to be null, if app.UseStaticFiles() wasn't included. We should check all references for null to be on the safe side, so we don't cause runtime errors in working applications.